src/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rentals.book = self
+  end
```

Comment:
- [ ] 2 parameters are needed by the Rental class namely:  the `date`, and the `person` renting the book. Please put that into consideration in your `add_rental` method 👍 

- [ ] There is really no need to manually add rentals to the `@rentals` array because this should already be handled whenever a new instance of the `Rental` is created.
Simply pass in the `date`, `book`, and `person` to the Rental class like this: 👍 
```ruby
Rental.new(date, self, person)
```